misc/config_tools/data/ehl-crb-b/hybrid_rt_launch_1uos_waag.xml

```diff
@@ -5,7 +5,7 @@
 	<mem_size desc="UOS memory size in MByte">4096</mem_size>
 	<gvt_args desc="GVT settings. Set it to gvtd for GVTd, otherwise is GVTg arguments. The recommendation arguments for GVTg is 64 448 8. Leave it blank to disable GVT.">gvtd</gvt_args>
 	<vbootloader desc="virtual bootloader method" readonly="true">ovmf</vbootloader>
-	<vuart0 desc="vUART0 which emulated by device model">Enable</vuart0>
+	<vuart0 desc="vUART0 which emulated by device model">Disable</vuart0>
```

Comment:
Shouldn't all these XML files with ``desc="..."`` in them be replaced by XML files based on the scenario schema definition (such as ``hybrid_rt_fusa.xml``)?